### PR TITLE
simpler C#.

### PR DESCRIPTION
**Pull request by Anduin2017**

### CODE REVIEW

File: Kahla.Server/Services/EmailNotifier.cs [hunk 1]
```diff
@@ -89,19 +89,14 @@ public async Task<string> BuildEmail(KahlaUser user, KahlaDbContext dbContext, I
...
-                    continue;
-                }
-
-                if (inConversations == 50)
+                if (inConversations >= 50)
                 {
-                    msg.AppendLine(
-                        "<li>Some conversations haven't been displayed because there are too many items.</li>");
-                    continue;
+                    msg.AppendLine("<li>Some conversations haven't been displayed because there are too many items.</li>");
```
Comment (hv0905) [inline Kahla.Server/Services/EmailNotifier.cs:98]: you can’t break the loop here or the convasations count won’t be right...

**Comment by Anduin2017** [comment]
@hv0905  What about now?

**Comment by hv0905** [comment]
What happened...

**Comment by hv0905** [comment]
It works nice, well done.

**Pull request merged**